pkg/instancegroups/rollingupdate_test.go

```diff
@@ -222,17 +222,14 @@ func TestRollingUpdateAllNeedUpdate(t *testing.T) {
 		case testingclient.PatchAction:
 			if string(a.GetPatch()) == cordonPatch {
 				assertCordon(t, a)
-				assert.Equal(t, "", cordoned, "at most one node cordoned at a time")
```

Comment:
I'm wondering if it would be worthwhile to hook into the `TerminateInstances()` mock in order to keep all of the `testingclient.DeleteAction` assertions.

Reply:
I thought about it, but it _shouldn't_ actually delete any nodes, so seems strange to test for it.
We are also checking that the ASGs are empty at the end, so we know the terminations take place.